SageMaker/01_introduction.ipynb

```diff
@@ -0,0 +1,1122 @@
+{
```

Comment:
<p>Nit: I suggest using <code>%pip install ...</code> over <code>!pip install</code> the dependencies are install in the correct environment. Also, you can put <code>%%capture</code> at the top of the cell if you want to suppress all the outputs (just an idea) </p><p> </p><p>As side remark: is there a reason to use the <code>pip</code> command instead of the <code>install_requirements()</code> function we created for Colab / Kaggle?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nlp-with-transformers/notebooks/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>done.</p><p>there are some dependencies which are hard to resolve (like path etc.). </p><p>it's not that easy to make it work with install_requirements() :-) </p>